examples/asr/conf/conformer/conformer_ctc_bpe.yaml

```diff
@@ -143,6 +143,15 @@ model:
     num_classes: -1
     vocabulary: []
 
+  # config for InterCTC loss: https://arxiv.org/abs/2102.03216
+  # specify loss weights and which layers to use for InterCTC
+  # e.g., to reproduce the paper results, set loss_weights: [0.3]
+  # and apply_at_layers: [9] (assuming 18 layers). Note that final
+  # layer loss coefficient is automatically adjusted (to 0.7 in above example)
```

Comment:
Would you please add this section to the fastconformer and cache-aware configs too?